rpa-builder/6.1/modules/ROOT/pages/toolbox-web-automation-web-session-firefox.adoc

```diff
@@ -48,35 +48,40 @@ using the Firefox profile manager.
 
 == Inbound Variables
 
-* Profile name
-* Prefer 64Bit Webdriver
-* Alternative user agent string
-* Accept untrusted certificates
-* Enable native events
-* Port
-* Web Session ID
-* Initial URL
+* *Profile name*
+* *Prefer 64Bit Webdriver*
+* *Alternative user agent string*
+* *Accept untrusted certificates*
+* *Enable native events*
+* *Port*
+* *Web Session ID*
+* *Initial URL*
 
 == Outbound Variables
 
 This action step has no outbound variables.
 
 // Wizard
-include::partial$toolbox-web-automation-web-session.adoc[tag=wizardCommon]
+include::partial$toolbox-web-automation-web-session.adoc[tags=wizardCommon;!pageLoadStrategy]
 
-*Browser specific settings*
+=== Browser specific settings
 
 image:toolbox-variable-web-automation-web-session-images/image6.png[An image showing the browser specific settings]

```

Comment:
Though it is an alt text, it would be great to add the full stop at the end of the sentence to improve the pronunciation of the speech machine.